### ISSUE TITLE
verbose logging dumps out function declarations

### ISSUE THREAD

**Issue by pdehaan** [mozilla/chronicle#103]
When running the server, `npm start`, I'm seeing some pretty funky function declarations in the output. Not sure if this could mess up Heka logging or anything downstream...

```
chronicle.server.db.db.VERBOSE: db.getUserById called. arguments are: a6c9411418f94710a712ab97e7a3541c, function (err, result) {
        log.verbose('does user exist? result is ' + JSON.stringify(result));
        if (err) {
          return cb(err, false);
        }
        cb(null, !!result, fxaId);
      }
chronicle.server.index.VERBOSE: does user exist? result is {"fxaId":"a6c9411418f94710a712ab97e7a3541c","email":"fake_user@example.com","oauthToken":"fakeOauthToken"}
chronicle.server.db.db.VERBOSE: db.updateVisit called. arguments are: a6c9411418f94710a712ab97e7a3541c, 8c5dbe7e-e650-47e9-a6e8-ed810381d9ad, function (err) {
        if (err) {
          log.warn(err);
          return reply(Boom.create(500));
        }
        reply();
      }
```

If I'm chasing the code correctly, it's coming from the `verbose()` method in [/server/db/db.js:17-20](https://github.com/mozilla/chronicle/blob/586c5f29979590d8e8ecb463d253edca57587cbf/server/db/db.js#L17-L20):

```js
function _verbose(funcName, funcArgs) {
  var args = funcArgs && [].join.call(funcArgs, ', ');
  log.verbose(funcName + ' called. arguments are: ' + args);
}
```

Not sure if we should try filtering the arguments and remove any functions or see if we can call `JSON.stringify(args)` (although that may convert functions into `null` which may be misleading/confusing).

**Comment by pdehaan** [comment]
Meh... One possible solution is:
```js
function _verbose(funcName, funcArgs) {
  var args = [].slice.call(funcArgs).map(function (arg) {
    if (typeof arg === 'function') {
      return '[Function ' + (arg.name || 'anonymous') + ']';
    }
    return arg;
  });
  log.verbose(funcName + ' called. arguments are: ' + JSON.stringify(args));
}
```

... which gives us the following output:
```
chronicle.server.db.db.VERBOSE: db.getUserById called. arguments are: ["a6c9411418f94710a712ab97e7a3541c","[Function anonymous]"]
chronicle.server.index.VERBOSE: does user exist? result is {"fxaId":"a6c9411418f94710a712ab97e7a3541c","email":"fake_user@example.com","oauthToken":"fakeOauthToken"}
chronicle.server.db.db.VERBOSE: db.getVisits called. arguments are: ["a6c9411418f94710a712ab97e7a3541c",25,"[Function onResults]"]
```

**Comment by nchapman** [comment]
> which gives us the following output

That's pretty. @6a68 how does this look to you?

**Comment by 6a68** [comment]
Yeah, it's no bueno. Right now, you can set log level from 'verbose' to 'info' in config/local.json. Really soon, #113 will land, in which I specifically enumerate the arguments I want to log, instead of literally logging `arguments`.

**Comment by 6a68** [comment]
¯\_(ツ)_/¯

Of course, we could also use @pdehaan's clever lil function, which is rather nice. Maybe we can put it inside the logger file & export it, or make it a wrapper around `log.verbose`?

**Comment by 6a68** [comment]
fixed by #113
